README.md

```diff
@@ -1,27 +1,70 @@
 # Jupyter Qt Console
 
+[![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/jupyter)
+[![Build Status](https://travis-ci.org/jupyter/qtconsole.svg?branch=master)](https://travis-ci.org/jupyter/qtconsole)
+[![Documentation Status](https://readthedocs.org/projects/qtconsole/badge/?version=stable)](http://qtconsole.readthedocs.org/en/stable/?badge=stable)
+
 A rich Qt-based console for working with Jupyter kernels,
 supporting rich media output, session export, and more.
 
+This is a very lightweight widget that largely feels like a terminal, but
```

Comment:
I think we should use `application` instead of `widget` here because `qtconsole` is really an application.